react-frontend/src/HOC/withLayout.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import Topnav from "Topnav";
```

Comment:
Why is `Topnav` component placed directly in `src` directory (and not inside `components`)?